### DISCUSSION TITLE
zh/datasets/detect/coco/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#21641] [Docs]
# zh/datasets/detect/coco/

探索用于物体检测和分割的 COCO 数据集。了解其结构、用途、预训练模型和主要特征。

https://docs.ultralytics.com/zh/datasets/detect/coco/

**Comment by UltralyticsAssistant** [comment]
👋 Hello, thank you for opening this discussion and for your interest in Ultralytics 🚀! This is an automated response to help you get started quickly—an Ultralytics engineer will follow up shortly.

We recommend visiting the Docs for comprehensive guidance, including many Python and CLI examples, where common questions are often already answered:
- Docs: https://docs.ultralytics.com/
- Python Usage: https://docs.ultralytics.com/usage/python/
- CLI Usage: https://docs.ultralytics.com/usage/cli/

If this is a 🐛 Bug Report, please provide a minimum reproducible example (MRE) to help us investigate efficiently:
- MRE Guide: https://docs.ultralytics.com/help/minimum-reproducible-example/

If this is a custom training ❓ Question, please include as much detail as possible (dataset samples, training logs, exact commands, environment details), and verify you are following our best practices:
- Tips for Best Training Results: https://docs.ultralytics.com/guides/model-training-tips/

Join the Ultralytics community where it suits you best:
- Discord for real-time chat 🎧: https://discord.com/invite/ultralytics
- Discourse for in-depth discussions: https://community.ultralytics.com/
- Subreddit for community threads: https://www.reddit.com/r/Ultralytics/

## Upgrade

Please upgrade to the latest ultralytics package with all requirements in a Python>=3.8 environment and PyTorch>=1.8 to ensure you’re testing with the newest fixes and features:
```bash
pip install -U ultralytics
```
- Requirements: https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml
- Python: https://www.python.org/
- PyTorch: https://pytorch.org/get-started/locally/

## Environments

YOLO can run in any of the following up-to-date verified environments (with all dependencies including CUDA/CUDNN, Python and PyTorch preinstalled):
- Notebooks with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- Google Cloud Deep Learning VM. See GCP Quickstart Guide: https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/
- Amazon Deep Learning AMI. See AWS Quickstart Guide: https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/
- Docker Image. See Docker Quickstart Guide: https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/ <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all Ultralytics CI tests are currently passing. CI tests verify correct operation of all YOLO Modes and Tasks on macOS, Windows, and Ubuntu every 24 hours and on every commit.

P.S. For your specific topic on COCO for detection and segmentation, you can also review the localized page you shared for details on structure, usage, and pretrained models:
- COCO Dataset (ZH): https://docs.ultralytics.com/zh/datasets/detect/coco/ 📚
